RELEASE.md

```diff
@@ -0,0 +1,8 @@
+Release type: patch
+
+Mutable default values are frozen to support lists and objects.
+```python
+@strawberry.field
+def search(self, arg: List[int] = [], input: MyInput = {}) -> int:
```

Comment:
`input: MyInput = {}`

Should this be `input: MyInput = MyInput()`?

Reply:
That never worked historically, but could be a new feature.

I think literals should work because of the resemblance between GraphQL objects, JSON objects, and Python dicts.